crates/json-log/src/lib.rs

```diff
@@ -0,0 +1,26 @@
+#[cfg(debug_assertions)]
+pub use log::{debug, trace};
+#[cfg(debug_assertions)]
+pub use serde_json::json;
+
+#[macro_export(local_inner_macros)]
+macro_rules! json_debug {
+    ($($t:tt)*) => {
+        #[cfg(debug_assertions)]
+        debug!(
+            "{}",
+            json!($($t)*)
+        );
+    };
+}
+
+#[macro_export(local_inner_macros)]
+macro_rules! json_trace {
+    ($($t:tt)*) => {
+        #[cfg(debug_assertions)]
```

Comment:
Not `debug_assertion` as we might want to enable logging even in release builds, so a `logging`.